### ISSUE TITLE
execute all source code lines in the tests of terralib

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [TerraME/terrame#1009]
Currently 78 lines of the source code are not being executed.

**Comment by pedro-andrade-inpe** [comment]
The remaining lines are:

Project.lua:132 (will be solved by #1317)

TerraLib.lua:
```
295, 296
319, 320
603
635
1558, 1559
1564
1568
1811
1886
2029
2030
```

Verify if some of these lines are already tested by other files. TerraLib file can be verified as if it was part of the C++ level. If it guarantees that everything in the other files of the package runs fine then it is ok.

**Comment by pedro-andrade-inpe** [comment]
(Wait for the merge of my pull request (part 1) to solve this issue.)
